semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/Steps.scala

```diff
@@ -86,6 +88,18 @@ class Steps[A](val raw: GremlinScala[A]) {
 
   def isEmpty: Boolean = !isDefined
 
+  /**
+    * Print help/documentation based on the current elementType `A`.
+    * Relies on all step extensions being annotated with @Traversal / @Doc
+    * Note that this works independently of tab completion and implicit conversions in scope - it will simply list
+    * all documented steps in the classpath
+    * */
+  def help()(implicit elementType: ClassTag[A]): String =
+    Help.renderTable(elementType.runtimeClass, verbose = false)
+
+  def helpVerbose()(implicit elementType: ClassTag[A]): String =
```

Comment:
Is this necessary and is this public? For now, I don't foresee users needing to see this when they hit `help<TAB>` and wanting to individually see traversal names. Could we maybe hide this in `.help.verbose` ?

Reply:
it's not 'necessary', but IMO it's useful to the user. 
no strong feelings though, we can drop it, wdyt @fabsx00 ?
alternative would be: `def help(verbose: Boolean = false)`